front/pages/auth/verify.vue

```diff
@@ -59,6 +63,28 @@ export default {
     this.login()
   },
   methods: {
+    async setDefaultAvatar() {
+      const userId = this.$store.getters['userData/userId'];
+      const resp = await fetch(`https://avatars.dicebear.com/api/bottts/${userId}.svg`)
```

Comment:
why do we even need to do this from client? Why not do that on server on user create ?

Reply:
We would have to make an HTTP request to an API that admits on the site that it isn't stable and could go down or become slow at any moment. My understanding is that the backend is billed based on the CPU time, right? So making HTTP requests to a third party API could become slow and expensive. Please correct me if this logic isn't sound. That was the basis of my decision here.

Reply:
Yes, but but if its not stable than maybe its not good idea to use at all. Because it will either imply wait for user at front or at backend. Also with current way it will set avatar on each login of user

Reply:
The API we use then is probably a question for @ezl . 

Also, my understanding is that it is set on FIRST login if the user hasn't yet been welcomed. Otherwise it does not set the avatar again. I have just tested this again and confirmed that is how it is working.

Reply:
weighing in here.

1. i think @aldarund is probably right that all other things equal, making the request from the backend is probably the best. if it's from the client, we're effectively making the request to the browser, then transmitting the image to the backend, so it's just an extra hop.

2. i think @f-prime is probably right that if the API is stated to be unstable or buggy, it would consume extra CPU time and risk  timeouts on the backend.

3. basically my guess is that it's STATED to be buggy so the author doesn't have to worry about supporting it. it's unclear how buggy or unreliable it is, if at all (my guess is it's fine)

4. the actual extra CPU cycles are probably negligible, even if it requires more AWS time.

5. "if it's not stable than maybe its not a good idea to use at all" - in an ideal world it would work perfectly. i don't think we have that option available to us, and i'd rather have something that works most of the time and provides a better site experience and sometimes fails, so this isn't an option.

I propose:

1. Merge this
2. Create a separate ticket for migrating this to the backend later (@f-prime can you create the ticket for moving the logic to the backend after account creation? Also potentialy create a ticket for gracefully handling total API failure (if it fails, just use a static image that we have))

Reasons:
1. Is it the optimal solution? Maybe not -- but an OPTIMAL solution may not exist given a stable supported API doesn't exist.
2. Is it better than nothing? Yes
3. Does it (in the happy path case) do what we want? Yes.

Does that work?

Reply:
i missed where its called, somehow i though its called in mounted in verify. weird. ye, it wont be called on each login.

Reply:
1 2 - fine for me but need comma eslint to be fixed